idseq_dag/steps/run_priceseq.py

```diff
@@ -61,3 +61,12 @@ def fq2fa(input_fastq, output_fasta):
         cmd = f"sed -n '1~4s/^@/>/p;2~4p' <{input_fastq} >{output_fasta}"
         command.execute(cmd)
         log.write(f"Finished {step}.")
+
+    @staticmethod
+    def multilinefa2singlelinefa(input_fasta, output_fasta):
+        ''' Multi-line FASTA to Single-line FASTA conversion '''
+        step = "Multi-line FASTA to single-line FASTA conversion"
+        log.write(f"Starting {step}...")
+        cmd = f"awk 'NR==1 {{print $0}} NR>1 && /^>/ {{printf(\"\\n%s\\n\",$0);next; }} NR>1 {{ printf(\"%s\",$0);}}  END {{printf(\"\\n\");}}' <{input_fasta} > {output_fasta}"
```

Comment:
If you added a comment on how to read the awk command, it would be super educational :) I actually wonder if there is not a simpler way to write the same command

Reply:
(namely
```
awk '{if(NR==1) {print $0} else {if($0 ~ /^>/) {print "\n"$0} else {printf $0}}}'
```
but no guarantees here)

Reply:
plus all the escaping

Reply:
yeah i copied/pasted someone else's code and then tweaked it rather than writing it from scratch. yours is more compact but less 'awk'-y